lib/ansible/plugins/terminal/exos.py

```diff
@@ -49,7 +52,11 @@ class TerminalModule(TerminalBase):
 
     def on_open_shell(self):
         try:
-            for cmd in (b'disable clipaging', b'configure cli columns 256'):
-                self._exec_cli_command(cmd)
+            self._exec_cli_command(b'disable clipaging')
         except AnsibleConnectionFailure:
             raise AnsibleConnectionFailure('unable to set terminal parameters')
+
+        try:
+            self._exec_cli_command(b'configure cli columns 256')
+        except AnsibleConnectionFailure:
+            display.display('WARNING: Unable to configure cli columns, command responses may be truncated')
```

Comment:
```suggestion
            self._connection.queue_message('warning', 'Unable to configure cli columns, command responses may be truncated')
```

And then remove the `Display` import above.

Reply:
Resolved and tested, works fine.